pkgs/applications/networking/instant-messengers/viber/default.nix

```diff
@@ -1,111 +1,167 @@
-{fetchurl, lib, stdenv, dpkg, makeWrapper,
- alsa-lib, cups, curl, dbus, expat, fontconfig, freetype, glib, gst_all_1,
- harfbuzz, libcap, libGL, libGLU, libpulseaudio, libxkbcommon, libxml2, libxslt,
- nspr, nss, openssl_1_1, systemd, wayland, xorg, zlib, ...
-}:
-
-stdenv.mkDerivation {
+{ pkgs, lib, buildFHSEnv }:
```

Comment:
```suggestion
{ lib, buildFHSEnv, xorg.xkeyboardconfig, open-fonts, xdg-utils }:
```

Reply:
Is this valid? `targetPkgs` seems to be a function taking `pkgs` as an argument. I believe this is to handle `x86_64` vs `x86`.

Reply:
Yeah, but in the end, `targetPkgs` is just called with the default `pkgs` as argument in https://github.com/NixOS/nixpkgs/blob/6722a8d3f9401652270abd8749fed6c8e6d478be/pkgs/build-support/build-fhsenv-bubblewrap/buildFHSEnv.nix#L45C87-L45C87.

Reply:
Yes it is a bit confusing, I assumed it was defined as a lambda for cross-compilation reasons. Since `pkgs` is provided as an argument, I thought I had to use the supplied `pkgs` argument's `callPackage` instead.